pkg/index/inverted/inverted.go

```diff
@@ -174,7 +183,7 @@ func (s *store) Iterator(fieldKey index.FieldKey, termRange index.RangeOpts, ord
 	if err != nil {
 		return nil, err
 	}
-	fk := fieldKey.MarshalIndexRule()
+	fk := fieldKey.Marshal()
```

Comment:
Please remove the line bluge.blugeMatchAllquery() from this function, as it's redundant.

Reply:
done